pynestml/generated/PyNestMLLexer.interp

```diff
@@ -0,0 +1,270 @@
+token literal names:
+null
```

Comment:
why is ``null`` repeated?

Reply:
No idea, this file is auto-generated by Antlr.